rpm/libgpg-error.spec

```
@@ -1,10 +1,9 @@
 Summary: Library for error values used by GnuPG components
 Name: libgpg-error
-Version: 1.27
+Version: 1.42
 Release: 1
 URL: ftp://ftp.gnupg.org/gcrypt/libgpg-error/
```

Can you change it to the actual one?
```suggestion
URL: https://www.gnupg.org/related_software/libgpg-error/
```

---

Updated also.